src/pages/[platform]/build-a-backend/auth/concepts/external-identity-providers/index.mdx

```diff
@@ -342,6 +342,93 @@ When you import and use the `signInWithRedirect` function, it will add a listene
 </Accordion>
 </InlineFilter>
 
+## Configure OIDC provider
+
+To setup a OIDC provider, you can configure them in your `amplify/auth/resource.ts` file. For example, if you would like to setup a Microsoft EntraID provider, you can do so as follows:
+
+```ts title="amplify/auth/resource.ts"
+import { defineAuth, secret } from '@aws-amplify/backend';
+
+export const auth = defineAuth({
+  loginWith: {
+    email: true,
+    externalProviders: {
+      oidc: [
+        {
+          name: 'MicrosoftEntraID',
+          clientId: secret('MICROSOFT_ENTRA_ID_CLIENT_ID'),
+          clientSecret: secret('MICROSOFT_ENTRA_ID_CLIENT_SECRET'),
+          issuerUrl: '<your-issuer-url>',
+        },
+      ],
+      logoutUrls: ['http://localhost:3000/', 'https://mywebsite.com'],
+      callbackUrls: [
+        'http://localhost:3000/profile',
+        'https://mywebsite.com/profile',
+      ],
+    },
+  },
+});
+```
+
+### Set up your frontend
+
+<InlineFilter filters={["angular", "javascript", "nextjs", "react", "vue"]}>
```

Comment:
can we link out to the sign-in page? these duplicate headings are going to cause issues with the anchor, and the inline filter will show an empty section

Reply:
https://github.com/aws-amplify/docs/pull/7566/files#diff-d59fe95d17fcc94afb4be025157ed7e88a6b2fb97f15755da89c6127e1b19c53L347 
but we do have it at already at end of page

can change to eg: setup SAML redirect on frontend